packages/studio/src/App.tsx

```diff
@@ -13,7 +21,15 @@ export default function App() {
         <div className="flex flex-row grow">
           <LeftSidebar />
           <div className="grow w-1/3 bg-white border-8 shadow">
-            <PreviewWithUseComponents />
+            <Tooltip
```

Comment:
Since these tooltips are only used for `PreviewWithComponents`, could we move the tooltip rendering logic into the `PreviewWithComponents` component?